tools/aa/commands/InitCommand.java

```diff
@@ -25,46 +25,68 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import javax.inject.Inject;
+import java.nio.file.Paths;
+import org.eclipse.jgit.api.Git;
 
+/** A command to init a base folder.
+ *
+ * Usage:
+ * bazel run //tools/aa:aa_tool -- init --base_path </path/to/base/folder>
+ * or, if aa is already set up:
+ * aa init --base_path </path/to/base/folder>
+*/
 public class InitCommand implements AaCommand {
-  @FlagDesc(name = "config_path", description = "Path to config file", required = true)
-  public static Flag<String> configPath = Flag.create("");
+  public static final String BASE_FILENAME = "BASE";
 
   @FlagDesc(name = "base_path", description = "Base path", required = true)
   public static Flag<String> basePath = Flag.create("");
 
-  @FlagDesc(name = "remote_repo_url", description = "Remote git repo", required = true)
-  public static Flag<String> remoteRepoUrl = Flag.create("");
+  @FlagDesc(name = "startupos_repo", description = "StartupOS git repo")
+  public static Flag<String> startuposRepo = Flag.create("git@github.com:google/startup-os.git");
 
   @FlagDesc(name = "user", description = "User")
   public static Flag<String> user = Flag.create(System.getenv("USER"));
```

Comment:
Maybe, do **basePath, startuposRepo, user variables** as final?

Reply:
Technically you're right, they can be final, but I find it kind of confusing that final fields can be changed if they're objects (i.e the reference cannot change, but the pointed object can).
For this reason, I tend to put final only when it means that nothing can change about the field - i.e the type is immutable.